examples/package.json

```diff
@@ -1,18 +1,23 @@
-
 {
   "name": "react-globalize-example",
+  "version": "0.4.1",
   "description": "Examples demonstrating use of the components",
   "main": "index.js",
   "devDependencies": {
-    "browserify": "^6.3.3",
-    "reactify": "^0.17.1"
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "imports-loader": "^0.7.1",
+    "webpack": "^3.5.4"
   },
   "dependencies": {
-    "cldr-data": "~26.0.9",
-    "react": "~0.13.1",
-    "globalize": "~1.0.0-alpha.18"
+    "cldr-data": ">=31",
+    "globalize": "^1.3.0",
+    "react": "^15.6.0",
+    "react-dom": "^15.6.1"
```

Comment:
Shall we add a `|| ^16.0.0`?

Reply:
You're right. Damned ;) I also forgot to add my new fav example with markup ;)